### PR TITLE
Settings gui

### PR DESCRIPTION
**Pull request by halfxblood**

### CODE REVIEW

File: src/main/java/gui/GUIDriver.java [hunk 1]
```diff
@@ -3,10 +3,7 @@
 import com.sun.source.tree.BreakTree;
 import gui.utility.InstanceMapper;
 import gui.utility.NavigationHelper;
-import gui.view.AddTaskPageController;
-import gui.view.MonthlyCalendarController;
-import gui.view.TodoListPageController;
-import gui.view.WeeklyCalendarController;
+import gui.view.*;
```
Comment (jenci2114) [inline src/main/java/gui/GUIDriver.java:6]: Avoid wild card import

Comment (jenci2114) [reply src/main/java/gui/GUIDriver.java:6]: You could change your IDE setting so it doesn't automatically merge imports into a wild card import

File: src/main/java/gui/viewmodel/SettingsViewModel.java [hunk 2]
```diff
@@ -0,0 +1,4 @@
+package gui.viewmodel;
+
+public class SettingsViewModel extends ViewModel{
+}
```
Comment (jenci2114) [inline src/main/java/gui/viewmodel/SettingsViewModel.java:4]: Do we need this class since it has no content?

Comment (valyippee) [reply src/main/java/gui/viewmodel/SettingsViewModel.java]: xander will be using it for ics exports

File: src/main/java/gui/GUIDriver.java (hunk 1 above)
Comment (valyippee) [reply src/main/java/gui/GUIDriver.java]: edited!

**Pull request merged**